libshaderc_util/src/spirv_tools_wrapper.cc

```diff
@@ -108,6 +109,7 @@ bool SpirvToolsAssemble(Compiler::TargetEnv env,
 bool SpirvToolsOptimize(Compiler::TargetEnv env,
                         Compiler::TargetEnvVersion version,
                         const std::vector<PassId>& enabled_passes,
+                        spvtools::OptimizerOptions& optimizer_options,
```

Comment:
In the past we've stayed away from non-const references as params.
But that rule is gone.
Fortunately the caller destroys the enclosing object pretty quickly, so this seems ok.